### PR TITLE
Hotfix 1.0.1

### PR DESCRIPTION
**Pull request by jenniferboedker**
Fixes security issue for

* log4j: 2.11.0 -> 2.15.0
* vaadin: 7.7.8 -> 7.7.24

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -98,12 +113,14 @@
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>vaadin-client</artifactId>
-            <scope>provided</scope>
+            <version>${vaadin.version}</version>
+            <scope>compile</scope>
```
Comment (KochTobi) [inline pom.xml]: why the scope change?

**Review by wow-such-code — APPROVED** [review]
lgtm

**Review by KochTobi — CHANGES_REQUESTED** [review]
scope change?

File: pom.xml (hunk 1 above)
Comment (jenniferboedker) [reply pom.xml]: ah my bad, needed it to compile it locally

**Review by KochTobi — APPROVED** [review]

**Pull request merged**